packages/core/src/layout/Header/Header.tsx

```diff
@@ -17,12 +17,13 @@
 import React, { Fragment, ReactNode, CSSProperties, FC } from 'react';
 import Helmet from 'react-helmet';
 import { Typography, Tooltip, makeStyles } from '@material-ui/core';
+import { BackstageTheme } from '@backstage/theme';
+
 import { Theme } from '../Page/Page';
 // import { Link } from 'shared/components';
-import { BackstageTheme } from '../../theme/theme';
 import Waves from './Waves';
 
-const useStyles = makeStyles<BackstageTheme>(theme => ({
+const useStyles = makeStyles<typeof BackstageTheme>(theme => ({
```

Comment:
the good news in this PR: you can use Typescripts' `typeof` to get a nice little bit of streamlined behavior here.